DNM: release 0.33.2

zakisk opened the pull request:
# Changes <!-- 🎉🎉🎉 Thank you for the PR!!! 🎉🎉🎉 -->
release 0.33.2 test 

https://issues.redhat.com/browse/SRVKP-7522

zakisk committed:
Invalidate ok-to-test command on pushed commits

invalidated ok-to-test command on pushed commits as it is not
needed there.

Signed-off-by: Zaki Shaikh <zashaikh@redhat.com>

zakisk committed:
Fix: PLR Definition Issue When Provenance is Set in GitLab

Issue: In GitLab, when provenance is set to default_branch,
the API call to get .tekton directory tree is correct and fetching
the tree as per provenance but when each is file is fetched as raw
its using `source` as provenance no matter what value has been set
in Repository CR.

Solution: This pull request use revision correctly in concatAllYamlFiles
func to make it obey provenance settings.

Signed-off-by: Zaki Shaikh <zashaikh@redhat.com>

zakisk committed:
fix: PAC controller crash on empty changes in BDC

fixed crash on push when there are no changes in
in event payload in Bitbucket Data Center.

https://issues.redhat.com/browse/SRVKP-7522

Signed-off-by: Zaki Shaikh <zashaikh@redhat.com>

zakisk commented:
/test linters

zakisk commented:
@chmouel docker image job was successful in your PR #2054 but failing here while both have same workflow file 🤔

chmouel commented:
@zakisk it's a bug, but no worries we can skip it, i think it's because we are doing a push and a pull_request, (same issues we have on pac reallly)

zakisk committed:
fix: Delete Gitea Cluster Task from E2E

deleted gitea cluster task e2e tests as clustertask
is deprecated in OSP and its CRDs will be removed in
upcoming release.

Signed-off-by: Zaki Shaikh <zashaikh@redhat.com>

savitaashture commented:
/lgtm

savitaashture commented:
Merging as per the discussion with @zakisk
